### DISCUSSION TITLE
Is there a way to set styles based on dynamic type sizes, rather than points?

### DISCUSSION THREAD

**Discussion by ryanbooker** [gonzalezreal/swift-markdown-ui#321] [Q&A]
Is there a way to set styles based on dynamic type sizes, rather than points? e.g. just using things like `body`, `footnote`, etc for standard line spacing, font sizes, paragraph spacing etc.

**Comment by gonzalezreal** [comment]
You can use sizes relative to the current or root font size. See:
https://swiftpackageindex.com/gonzalezreal/swift-markdown-ui/2.3.0/documentation/markdownui/relativesize

FYI, the [Basic](https://github.com/gonzalezreal/swift-markdown-ui/blob/c0daf6eb79d97964180f3113868c990bd1c4a007/Sources/MarkdownUI/Theme/Theme%2BBasic.swift) and [DocC](https://github.com/gonzalezreal/swift-markdown-ui/blob/main/Sources/MarkdownUI/Theme/Theme%2BDocC.swift) themes use relative sizes.

**Reply by ryanbooker** [reply]
Ah cool… the root size being say something set via `.font(.body)` on a view?

**Reply by gonzalezreal** [reply]
The root size being the size of the theme's default text style, which, if not explicitly set, is equivalent to the system's body font size.
